controllers/storagecluster/exporter.go

```diff
@@ -709,3 +715,94 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+// rookCephMonitorMgrRoleBindingJSON rolebindings for rook ceph monitor
+const rookCephMonitorMgrRoleBindingJSON = `
+{
+	"apiVersion": "rbac.authorization.k8s.io/v1",
+	"kind": "ClusterRoleBinding",
+	"metadata": {"name":"rook-ceph-monitor-mgr"},
+	"roleRef": {
+		"apiGroup": "rbac.authorization.k8s.io",
+		"kind": "ClusterRole",
+		"name": "rook-ceph-monitor-mgr"
+	},
+	"subjects": [
+		{
+			"kind": "ServiceAccount",
+			"name": "rook-ceph-mgr"
+		}
+	]
+}`
+
+// rookCephMonitorRoleBindingJSON rolebindings for rook ceph monitor
+const rookCephMonitorRoleBindingJSON = `
+{
+	"apiVersion": "rbac.authorization.k8s.io/v1",
+	"kind": "ClusterRoleBinding",
+	"metadata": {"name":"rook-ceph-monitor"},
+	"roleRef": {
+		"apiGroup": "rbac.authorization.k8s.io",
+		"kind": "ClusterRole",
+		"name": "rook-ceph-monitor"
+	},
+	"subjects": [
+		{
+			"kind": "ServiceAccount",
+			"name": "rook-ceph-system"
+		}
+	]
+}`
+
+func createRookCephRolebindings(ctx context.Context,
```

Comment:
Please rebase onto latest main and follow the same pattern to create clusterrolebindings.
We are not using JSON anymore and create/update logic is straightforward.

Reply:
Done